examples/adminapi.py

```diff
@@ -917,6 +950,7 @@ def example_delete_records(a, args):
         sys.stderr.write(' list_offsets <isolation_level> <topic1> <partition1> <offset_spec1> ' +
                          '[<topic2> <partition2> <offset_spec2> ..]\n')
         sys.stderr.write(' delete_records <topic1> <partition1> <offset1> [<topic2> <partition2> <offset2> ..]\n')
+        sys.stderr.write(' elect_leaders <election_type> <topic1> <partition1> [<topic2> <partition2> ..]\n')
```

Comment:
Now all topic partition are optional. So change the usages accordingly.

```suggestion
        sys.stderr.write(' elect_leaders <election_type> [<topic1> <partition1> <topic2> <partition2> ..]\n')
```